docs/markdown/Reference-manual.md

```diff
@@ -547,11 +547,12 @@ libraries (`.so`, `.dll`, etc) will be linked.
 
 With the Ninja backend, Meson will create a build-time [order-only
 dependency](https://ninja-build.org/manual.html#ref_dependencies) on
-all generated input files, including unknown files. For all input
-files (generated and non-generated), Meson uses the [dependency
-file](https://ninja-build.org/manual.html#ref_headers) generated by
-your compiler to determine when to rebuild sources. The behavior is
-similar for other backends.
+all generated input files, including unknown files. This is needed
+to bootstrap the generation of the real dependencies in the
+[depfile](https://ninja-build.org/manual.html#ref_headers)
+generated by your compiler to determine when to rebuild sources.
+Meson relies on this dependency file for all input files, generated
```

Comment:
I know this that the original says this, but I 'm pretty sure that `meson` is supposed to be `ninja`

Reply:
I meant "meson _indirectly_ relies on..." based on the last sentence which is: "The behavior is similar for other backends."

I can either add the missing "indirectly" or change to ninja, whatever works.

Reply:
I thinks `meson` ... `other backends`. is confusing, as meson is not a backend, ninja is. I'd be just as happy to drop the `other backends part`

Reply:
I (mis?)understand the "similar behavior for other **backends**" at the end of this paragraph to be referring to: "For the Ninja **backend**, Meson will create a build-time order-only dependency..." at the top of this paragraph.

On the other hand, you seem to (mis?)understanding this "similar behavior for other backends" to be referring to the "Meson relies on..." sentence immediately before that I'm modifying, correct?

My modification admittedly creates a little more distance between the top and end of this paragraph.

The git log traces this paragraph back to... a mass import from the wiki in 2017 :-(

Reply:
This was most likely originally written by me and it was wrong, it should have said Ninja.